astro.config.mjs

```
@@ -4,10 +4,12 @@ import clerk from '@clerk/astro'
 import icon from 'astro-icon'
 import react from '@astrojs/react'
 import sitemap from '@astrojs/sitemap'
+import vercel from '@astrojs/vercel';
 
 export default defineConfig({
   site: 'https://pixelatedempathy.com',
   integrations: [clerk(), icon(), react(), sitemap()],
+  adapter: vercel(),
```

Biome (1.9.4)

[error] This property value named adapter is later overwritten by an object member with the same name.

Overwritten with this value.

If an object property with the same name is defined multiple times (except when combining a getter with a setter), only the last definition makes it into the object and previous definitions are ignored.
Unsafe fix: Remove this property value named adapter


(lint/suspicious/noDuplicateObjectKeys)